### ISSUE TITLE
PHP 8.x.x update?

### ISSUE THREAD

**Issue by sviluppomania** [PI-Media/json-api#10]
??

@pimedia

**Comment by pimedia** [comment]
Plugin works with php version 8.
    On Sunday, January 9, 2022, 07:45:05 PM GMT+5, sviluppomania ***@***.***> wrote:  
 
 


??

@pimedia

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by pimedia** [comment]
works with php 8.

**Comment by sviluppomania** [comment]
@pimedia 
Upgrading PHP version 7.4 to 8.0 I got a critical error: A critical error has occurred on your website.

For example: /get_category_posts/? Slug=app-store&page=1

Consequently not showing me the articles of the Site in the Mobile App.

**Comment by pimedia** [comment]
Yes, there is an issue on this endpoint. will let you know later. 
    On Monday, January 10, 2022, 09:06:20 PM GMT+5, sviluppomania ***@***.***> wrote:  
 
 


@pimedia
Upgrading PHP version 7.4 to 8.0 I got a critical error: A critical error has occurred on your website.

For example: /get_category_posts/? Slug=app-store&page=1

Consequently not showing me the articles of the Site in the Mobile App.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by sviluppomania** [comment]
@pimedia  ?

**Comment by sviluppomania** [comment]
@pimedia Hi, is there a solution to this critical endpoints error?

**Comment by pimedia** [comment]
Yes, but i have been busy in soem other tasks, please give some for this php 8 compatibility. 
    On Friday, January 14, 2022, 01:43:06 AM GMT+5, sviluppomania ***@***.***> wrote:  
 
 


@pimedia Hi, is there a solution to this critical endpoints error?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by sviluppomania** [comment]
@pimedia Hi, sorry but I don't understand.

**Comment by pimedia** [comment]
php 8 compatibility is too early to be available. It will need extensive testing.
    On Wednesday, January 19, 2022, 11:01:01 PM GMT+5, sviluppomania ***@***.***> wrote:  
 
 


Hello @pimedia ,
we look forward to an update of the Plugin for stable compatibility with PHP version 8.0 and later.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by sviluppomania** [comment]
> version 2.2 fixes the php 8 issues.

@pimedia Thanks a lot, now it works! :-)
I have added other Pull requests, if you can please merge them. Thank you.
